vms/administrator/templates/administrator/view_report.html

```diff
@@ -0,0 +1,44 @@
+{% extends "vms/base.html" %}
+{% load i18n %}
+{% block content %}
+<div class="spacer"></div>
+{% if report.confirm_status is 0 %}
+<div class="well well-sm">
+   <a href="{% url 'administrator:approve_report' report.id %}" class="btn btn-success btn-sm">{% trans "Approve Report" %}</a>
+</div>
+{% endif %}
+<div class="well">
+   <legend>{% trans "Report Statistics" %}</legend>
+   <label>
+      <h4><span class="label label-default">{% trans "Total Hours" %}</span></h4>
+   </label>
+   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{{total_hours }}
```

Comment:
Doesn't this needs to be fixed as part of #746 ?

Reply:
Now that PR doesn't exist.